src/Components/DetailsPage/DetailsPage.css

```diff
@@ -0,0 +1,80 @@
+.details-container {
```

Comment:
Great start on the Details Page Styling!

Reply:
Really great job! I love it!